zebra/rt_netlink.c

```diff
@@ -2713,6 +2723,18 @@ ssize_t netlink_nexthop_msg_encode(uint16_t cmd,
 							    ctx->table))
 							return 0;
 						break;
+					case SEG6_LOCAL_ACTION_END_DT46:
```

Comment:
I see no code added to decode these types of nexthops from the kernel?

Reply:
The code to decode `SEG6_LOCAL_ACTION_END_DT46` nexthops is already present in ``parse_encap_seg6local()``. The `SEG6_LOCAL_ACTION_END_DT46` uses the Netlink attribute `SEG6_LOCAL_VRFTABLE` which is already supported for the `SEG6_LOCAL_ACTION_END_DT4` nexthop.